omod/src/main/webapp/resources/scripts/field/personRelationship.js

```diff
@@ -74,4 +74,9 @@ angular.module('personRelationships', ['personService', 'relationshipService', '
                 }).join(', ');
             }
         }
+        field.value = function() {
```

Comment:
LGTM  thanks for working on this, however you may encounter merge conflicts, so ideally if this code is merged first before[ RA-1855](https://issues.openmrs.org/browse/RA-1855), then we can go ahead and resolve conflicts thanks alot
cc @cioan  @dkayiwa,